### ISSUE TITLE
Jumping panel title + empty space

### ISSUE THREAD

**Issue by spacegaier** [home-assistant/frontend#6985]
## Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

## The problem

When switching to another panel, the title is jumping to the right since the hidden button space gets added. 

Can't we get rid of the hidden empty icon space (`<ha-menu-button>`), if a separate side bar is shown, since then the drawer / burger icon is hidden anyway but still takes up space (which looks wrong independent of the jumping)? So perhaps some dynamic overlay CSS magic that if the screen is wide enough to display the sidebar, then `ha-menu-button` allows the title element to overlay it?

![B6F29966-5BDA-4292-B73E-F612C0173B32](https://user-images.githubusercontent.com/114137/93078447-c8065780-f68a-11ea-8870-9845aa442639.GIF)

## Expected behavior

1. No jumping title
2. No empty space in front of the title (or perhaps show the panel icon from the sidebar again, but I did not see an easy way to get access to the icon code other than hard-coding / duplicating it in the panel code itself which feels wrong)

## Environment

- Home Assistant release with the issue: latest dev (0.115)
- Last working Home Assistant release (if known):
- Browser and browser version: FF and Chrome
- Operating system: Win 10

## State of relevant entities

```yaml

```

## Problem-relevant configuration

```yaml

```

## Javascript errors shown in your browser console/inspector

```txt

```

**Comment by spacegaier** [comment]
@bramkragten This issue still persists even with the merged PR. I still see the jumping titles.

**Comment by maykar** [comment]
I do notice the quick jumping of titles now, my apologies. Though the empty space should no longer be an issue.

**Comment by spacegaier** [comment]
@maykar Yes, the empty space is gone 👍 , only the jumping is left from this issue.

**Comment by maykar** [comment]
Looks like the menu button isn't the cause of the issue here then, I suspect app-header, but will look into it further.

**Comment by spacegaier** [comment]
Still an issue in latest dev.

@maykar Did you have a chance to look further into this?

**Comment by maykar** [comment]
Unfortunately not, I've been MIA for a while. Just recently started working on my own projects again and am looking to start contributing here again soon.

**Comment by spacegaier** [comment]
Still an issue in 2021.4.4 and latest dev.

**Comment by spacegaier** [comment]
Issue still persists on latest dev
